code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -26,22 +26,32 @@
 	emote_hear = list("rurrs", "rumbles", "rowls", "groans softly", "murrs", "sounds hungry", "yawns")
 	emote_see = list("stares ferociously", "snarls", "licks their chops", "stretches", "yawns")
 	say_maybe_target = list("Ruh?", "Waf?")
-	say_got_target = list("Rurrr!", "ROAR!", "MINE!", "RAHH!", "Slurp.. RAH!")
+	say_got_target = list("Rurrr!", "ROAR!", "RERR!", "NOM!", "MINE!", "RAHH!", "RAH!", "WARF!")
 	melee_damage_lower = 5
 	melee_damage_upper = 20
 	response_help = "pets the"
 	response_disarm = "bops the"
 	response_harm = "hits the"
 	attacktext = "mauled"
-	friendly = list("nuzzles", "slobberlicks", "noses softly at", "noseboops")
-	meat_amount = 5
+	friendly = list("nuzzles", "slobberlicks", "noses softly at", "noseboops", "headbumps against", "leans on", "nibbles affectionately on")
+	meat_amount = 6
 	old_x = -16
 	old_y = 0
 	pixel_x = -16
 	pixel_y = 0
 
 	var/mob/living/carbon/human/friend
 	var/tamed = 0
+	var/list/friendmobs = list(
```

Comment:
There's a much more effective way to handle this, which is currently being resolved in https://github.com/VOREStation/VOREStation/commit/4367bc2da762a7acf5316e7cfcfadf6be5172485

Can you discard these changes?